module-4/backend/week_two.md

```diff
@@ -5,17 +5,29 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is Webpack and why is it useful?
+Webpack is a compiler that converts all JS to ES5 for browser compatability & compiles & minifies js, css, and html.
 2. When do you want to use event delegation?
+Event delegation is used to listen for events on a higher "bubble" than the actualy event; you can listen to something on the "outside" & delegate tasks according to the event's target.
```

Comment:
👍 it's also useful when you're looking to bind an event to an element that isn't on the page on page load